testing/functional/tests/cypress/page-objects/student-objects/02-login-object.ts

```diff
@@ -0,0 +1,85 @@
+class Login {
```

Comment:
I would say that in general, we have been declaring the classes as below:
```
export default Login {
```

Reply:
Yes, changes have been done. declared classes as mentioned above.